gapic/src/main/com/google/gapid/perfetto/views/RootPanel.java

```diff
@@ -466,6 +481,15 @@ protected void preMainUiRender(RenderContext ctx, Repainter repainter) {
       }
     }
 
+    private void renderFlags(RenderContext ctx) {
+      state.getFlags().forEach(e -> {
```

Comment:
Maybe use a more meaningful 't' instead of 'e' as the variable name. 'e' is used in lambdas across gapid because they means "events" in those contexts.